### PR TITLE
Fix canvas rooms in studies.

### PR DESCRIPTION
**Pull request by kaj**
Actually filter the canvas rooms, rather than just showing whatever one happens to first in the json.

### CODE REVIEW

File: kpm/kpm-frontend/src/panes/studies.tsx [hunk 1]
```diff
@@ -131,10 +131,12 @@ function RoundDesc({ round }: { round: TStudiesCourseRound }) {
 
 function Course({ courseCode, course }: TCourseProps) {
   const [roundToShow, setRoundToShow] = React.useState(course.rounds?.[0]);
-  // TODO: Try to find relevant room for selected round?
-  const roomToShow = course.rooms?.[0] || undefined;
   const status = course.completed ? "godkand" : roundToShow?.status;
   const exams = course.rooms?.filter((c) => c.type === "exam");
+  const selTerm = `${roundToShow.year}${roundToShow.term}`;
+  const filteredRooms = course.rooms?.filter(
+    (c) => c.type !== "exam" && (!c.startTerm || c.startTerm === selTerm)
```
Comment (jhsware) [inline kpm/kpm-frontend/src/panes/studies.tsx:138]: Perhaps add a note that !c.startTerm is used so we don't hide course rooms without terms (it isn't obvious unless one has had our discussion).

**Review by jhsware — CHANGES_REQUESTED** [review]
Minor

File: kpm/kpm-frontend/src/panes/studies.tsx (hunk 1 above)
Comment (kaj) [reply kpm/kpm-frontend/src/panes/studies.tsx:139]: Fixed.  Ok?

**Review by jhsware — APPROVED** [review]
LGTM

**Pull request merged**